README.md

```diff
@@ -47,16 +47,19 @@ Now you're ready to install Mist:
     $ cd mist
     $ git submodule update --init
     $ npm install
-    $ gulp update-nodes --platform all
+    $ gulp update-nodes
 
 To update Mist in the future, run:
 
     $ cd mist
     $ git pull && git submodule update
     $ npm install
-    $ gulp update-nodes --platform all
+    $ gulp update-nodes
 
 
+#### Options
+it is possible to pass the platform will be installed, see more [here](#platform)
```

Comment:
To stay consistent with the overall appearance you can also may want to change it to a sentence with punctuation.
Feel free to incorporate any of this as you see fit:
>It may be preferable to only download platform-specific nodes by passing the `--platform` flag, please refer to the [*options* section](#platform).